builders/get-started/eth-compare/tx-fees.md

```diff
@@ -211,7 +211,7 @@ And then `TransactionWeight` is mapped to the following field of the block JSON
 extrinsics[extrinsic_number].events[event_number].data[0].weight
 ```
 
-### Key Differences with Ethereum {: #ethereum-api-transaction-fees}
+### Key Differences with Ethereum {: #key-differences-with-ethereum}
```

Comment:
I feel that this section should be at the top of the page as an overall Summary. Also MBIP-5 Should have its own section where we show a clear difference from Ethereum. Maybe do a simple contract example where there is a clear contrast in gas estimation on Ethereum and on Moonbeam?

Or maybe  a contract that does provide the same exact gas estiation for a function, but then one very dfferent for another function?